qiime2/core/testing/plugin.py

```diff
@@ -1110,6 +1111,32 @@ def factory():
                 ' collections'
 )
 
+dummy_plugin.methods.register_function(
+    function=random_seed_method,
+    inputs={},
+    parameters={
+       'random_seed': Capture[Int]
```

Comment:
Do we need a semantic type for this? I don't think it changes how any interface would interact with the action.

Reply:
I'm not saying no to it necessarily, but we will need to change every interface to ignore/unwrap that outer type. Is that worth the effort?